hgtector/analyze.py

```diff
@@ -276,7 +276,7 @@ def read_search_results(file, maxhits=None):
         data = []
         with read_file(file) as f:
             for line in f:
-                line = line.rstrip()
+                line = line.rstrip("\n")
```

Comment:
It works! Maybe change that `"\n"` into `'\r\n'` just to be consistent with the current coding style?

Reply:
done